astro-client/client.go

```diff
@@ -65,7 +65,7 @@ func (c *HTTPClient) prepareRESTRequest(doOpts *httputil.DoOptions) error {
 	if cl.Token != "" {
 		doOpts.Headers["authorization"] = cl.Token
 	}
-	doOpts.Path = cl.GetPublicRESTAPIURL() + doOpts.Path
+	doOpts.Path = cl.GetPublicRESTAPIURL("") + doOpts.Path
```

Comment:
Should this be some value and not an empty string?

Reply:
no it should be an empty string because in v1alpha one it's all one API. In v1beta1 this value is either platform or iam

Reply:
can we add that note as a comment in the code ^